hail_scripts/reference_data/combine.py

```diff
@@ -110,8 +110,8 @@ def get_ht(dataset: str, reference_genome: str):
     ht = ht.select_globals(
         **{
             f'{dataset}_globals': hl.struct(
-                path=config['path'],
-                version=parse_version(ht, config),
+                path=config['source_path'] if 'custom_import' in config config['path'],
```

Comment:
Do this need an else?
```suggestion
                path=config['source_path'] if 'custom_import' in config else config['path'],
```